plugins/angular.js

```diff
@@ -9,36 +9,27 @@ var wrappedCallback = require('../src/utils').wrappedCallback;
 var angularPattern = /^\[((?:[$a-zA-Z0-9]+:)?(?:[$a-zA-Z0-9]+))\] (.*?)\n?(\S+)$/;
 var moduleName = 'ngRaven';
 
+function ExceptionHandlerProvider($provide) {
```

Comment:
Could you separate this code from a PR into a new one? It's changing the internal code for the plugin (constant vs. provider), therefore it should be taken care of on its own.